packages/docs/src/routes/docs/(qwikcity)/action/index.mdx

```diff
@@ -275,6 +281,20 @@ export const useAddUser = routeAction$(
       lastName: z.string(),
     });
   })
+  // Valibot version
+  /*
+    valibot$((v, ev) => {
+      const firstName =
+          ev.url.searchParams.get("firstname") === "optional"
+          ? v.optional(v.string())
+          : v.string([v.minLength(1)])
+
+      return z.object({
```

Comment:
Perfect example right here. This comment is wrong as we are mixing both valibot and zod. Imagine the frustration of the person trying to use this. We need real working code not comments.

Reply:
Added separate sections for Valibot.